tests/test_parameter_interpolation.py

```diff
@@ -123,32 +124,29 @@ def u_reference(x, params, box, lamb):
             qlj = interpolate_params(lamb, qlj_src, qlj_dst)
             return ref_potential(x, qlj, box, lamb)
 
+        qlj = np.concatenate([qlj_src, qlj_dst])
         for precision, rtol, atol in [(np.float64, 1e-8, 1e-11), (np.float32, 1e-4, 1e-6)]:
 
-            for lamb in [0.0, 0.2, 0.6, 0.7, 0.8, 1.0]:
-
-                qlj = np.concatenate([qlj_src, qlj_dst])
-
-                print("lambda", lamb, "cutoff", cutoff, "precision", precision, "xshape", coords.shape)
-
-                args = copy.deepcopy(test_potential.args)
-                args.append("lambda*lambda")  # transform q
-                args.append("sin(lambda*PI/2)")  # transform sigma
-                args.append("lambda < 0.5 ? sin(lambda*PI)*sin(lambda*PI) : 1")  # transform epsilon
-                args.append("1-lambda*lambda")  # transform w
-
-                test_interpolated_potential = potentials.NonbondedInterpolated(
-                    *args,
-                )
-
-                self.compare_forces(
-                    coords,
-                    qlj,
-                    box,
-                    lamb,
-                    u_reference,
-                    test_interpolated_potential,
-                    rtol=rtol,
-                    atol=atol,
-                    precision=precision,
-                )
+            print("cutoff", cutoff, "precision", precision, "xshape", coords.shape)
+
+            args = copy.deepcopy(test_potential.args)
+            args.append("lambda*lambda")  # transform q
+            args.append("sin(lambda*PI/2)")  # transform sigma
+            args.append("lambda < 0.5 ? sin(lambda*PI)*sin(lambda*PI) : 1")  # transform epsilon
+            args.append("1-lambda*lambda")  # transform w
+
+            test_interpolated_potential = potentials.NonbondedInterpolated(
+                *args,
```

Comment:
nit:
```suggestion
                *args
```
(should fix weird formatting)